src/xSTIR/pSTIR/STIR.py

```diff
@@ -1627,22 +1627,40 @@ def get_time_at_which_num_prompts_exceeds_threshold(self, threshold):
 
 class PoissonNoiseGenerator(object):
     """
-    Class that generates Poisson noise.
+    Generates noise realisations according to Poisson statistics but allowing for scaling.
+
+    A scaling_factor is used to multiply the input data before generating
+    the Poisson random number. This means that a scaling_factor larger than 1
+    will result in less noisy data.
+
+    If preserve_mean=false, the mean of the output data will
+    be equal to <tt>scaling_factor*mean_of_input</tt>, otherwise it
```

Comment:
can't use html in Python docstrings, so just drop the tt stuff

Reply:
done